services/src/app.js

```diff
@@ -31,15 +31,29 @@ const sequelizeSwagger = require('./sequelize-swagger');
 const settings = configuration();
 const app = express(feathers());
 
+
 // const SUCCESS = 'OK';
 const FAILURE = 'ERROR';
 
 /*
-  * Allow the log level to be overridden in the environment for debugging
-  * purposes by the user
-  */
+ * Allow the log level to be overridden in the environment for debugging
+ * purposes by the user
+ */
 logger.level = process.env.LOG_LEVEL || 'warn';
 
+
+/*
+ * Configure Sentry integration only we have a SENTRY_DSN environment
+ */
+if (process.env.SENTRY_DSN) {
+  logger.info('Configuring Sentry for backend errors..');
+  const Raven = require('raven');
+  Raven.config(process.env.SENTRY_DSN).install();
+  app.use(Raven.requestHandler());
+  app.use(Raven.errorHandler());
+  logger.info('..Sentry middleware installed');
+}
+
```

Comment:
This is duplicating the configuration done further down by the sentry class, isn't it?

Reply:
Hm, you're right, damn you singeltons!